forest/sycamore/utils.py

```diff
@@ -0,0 +1,51 @@
+"""Utility functions in Sycamore.
+This file exists to avoid a circular imports from common.py.
+In the future, it would be good to refactor code and put more things in here,
+but for now, I'll just include the three necessary functions so this pull
+request isn't massive"""
+
+import datetime
+import json
+import re
+
+import pandas as pd
+
+
+def read_json(study_dir: str) -> dict:
+    """Read a json file into a dictionary
+
+    Args:
+        study_dir (str):  filepath to json file.
+    Returns:
+        A dict representation of the json file
+    """
+    with open(study_dir, "r") as f:
+        dictionary = json.load(f)
```

Comment:
Simply: `return json.load(f)`?

Reply:
You still need to open the file with a context manager to make sure the file is closed when processing is finished:
```python
with open(study_dir, "r") as f:
    return json.load(f)
```